extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -554,7 +560,15 @@ export class JupyterServerInstallation {
 		let packagesInfo = await this.executeBufferedCommand(cmd);
 
 		if (packagesInfo) {
-			let packagesResult = JSON.parse(packagesInfo);
+			let packagesResult = [];
+			try {
+				packagesResult = JSON.parse(packagesInfo);
+			}
+			catch (err) {
+				let errorMsg = `Failed to parse package info, encountered following error: ${(utils.getErrorMessage(err))} \n PackageInfo: ${packagesInfo}`;
```

Comment:
Could you make this a bit more descriptive? Maybe something like "Encountered an error when trying to retrieve list of installed packages: {0}". Just getting a parse failure message may be too vague. Also, could you refactor both of these into a single method up with the other localized messages?